pkg/sql/logictest/testdata/logic_test/alter_type

```diff
@@ -154,7 +154,7 @@ ALTER TYPE build ADD VALUE 'a' BEFORE 'b'
 statement error pq: \"b\" is not an existing enum label
 ALTER TYPE build ADD VALUE 'a' AFTER 'b'
 
-statement ok
+statement error pq: enum label \"c\" already exists, skipping
```

Comment:
Please look at the test I mentioned. It isn't correct to return an error here! The idea of a notice is that the statement executes successfully, but just prints out a notice to the user.

Reply:
this is pushed before i saw your reviews, i have corrected in current commit.